src/main/java/net/stickycode/plugin/shifty/ShiftyFetchMojo.java

```diff
@@ -115,8 +148,20 @@ Artifact lookupArtifact(DefaultArtifact artifact) {
       artifact.getClassifier(),
       artifact.getExtension(),
       version.toString());
-    ArtifactRequest request = new ArtifactRequest(fetch, repositories, null);
-    return resolve(request).getArtifact();
+    return new ArtifactRequest(fetch, repositories, null);
+  }
+
+  Properties getProjectProperties() {
+    return project.getProperties();
+  }
```

Comment:
Why?

Reply:
exposed for testing, should probably put the annotations or use Mockito